### PR TITLE
VP-1322 new user cannot save profile

### PR DESCRIPTION
**Pull request by bernesemountaindog**
## I do solemnly swear that I have:
- [x] Run `npm test` and all the tests pass.
- [x] Run `npm run lint` and there are no warnings.
- [x] Not decreased the overall test coverage?
- [x] Included the Jira ID and description in the PR title

## Proposed Changes? 🤔
1. After locking down the person API, frontend clients can no longer send the email, dateAdded field in their payload
2. 
3. 

## Additional Info.🧐

Any additional info goes here

### CODE REVIEW

File: components/Home/HomeTabs.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const HomeTabs = ({ person, onChange, defaultTab }) =>
     </TabPane>
     <TabPane tab={homeProfileTab} key='profile'>
       <ProfilePanel>
-        <EditablePersonPanel person={person} />
+        <EditablePersonPanel person={person} me={me} />
```
Comment (avowkind) [inline components/Home/HomeTabs.js]: it is not necessary to pass 'me' down through multiple levels of properties.  me is in the redux session so it is available everywhere. 
use the hook.
const me = useSelector(state => state.session.me)

Comment (bernesemountaindog) [reply components/Home/HomeTabs.js]: Sounds good I'll change this

Comment (bernesemountaindog) [reply components/Home/HomeTabs.js]: Looks like useSelector is only for NFC, PersonDetailForm is a class so it doesn't like being there
Anyway to access the redux store directly?

Comment (avowkind) [reply components/Home/HomeTabs.js]: export default connect(store => ({  me: store.session.me } )(PersonDetailForm)
should do it.

Comment (avowkind) [reply components/Home/HomeTabs.js]: Make the existing default export into a named const and wrap that.

Comment (bernesemountaindog) [reply components/Home/HomeTabs.js]: Thanks :) That's worked nicely :) I've only used redux a little bit, it solves this case of piping props down and down nicely :)

**Pull request merged**